src/engraving/dom/trill.cpp

```diff
@@ -246,6 +246,68 @@ void Trill::computeStartElement()
     }
 }
 
+PointF Trill::linePos(Grip grip, System** system) const
```

Comment:
Let's move out these two (seemingly identical?) functions into a single one. For instance, you could make it a static function of the Trill class, call it perhaps `static PointF Trill::trillLinePos`, let it take as input argument an `SLine*`, then both the `linePos` functions can call it (the one in the Vibrato class will have to refer to it as Trill::trillLinePos). 
Anyway, all of this is layout code, which should be moved out of the domain model and into the layout modules eventually. For now let's just try to avoid the repetition